lib/generateMarkdown.js

```diff
@@ -433,7 +431,7 @@ function getAnyOfEnumString(schema, type, depth) {
             enumString  += " " + enumDescription;
         }
 
-        allowedValues += prefix + enumString;
+        allowedValues += style.bulletItem(enumString, depth).toString();
```

Comment:
Why is the `toString()` necessary here?  Did you mean to put that on the `enumString` instead?